provision/acc_provision/acc_provision.py

```diff
@@ -1447,12 +1461,25 @@ def generate_kube_yaml(args, config, operator_output, operator_tar, operator_cr_
             new_deployment_file = temp
 
         if args.cko:
+            netopConfig = dict(config)
+            if "connectivity_checker" in netopConfig.keys():
```

Comment:
this code block is repeated, would it make sense to factor it out?

Reply:
addressed

Reply:
In the Installer CR we already have this check "{% if config.connectivity_checker %}", so nettools will only get populated if the input file has that config. Hence, we dont need the above check again.

Reply:
If we remove this check here, then all time (with cko arg and without connectivity_check) it will try to generate nettools deployment files which will fail because of nettools config does not exist. So we need this check here.